src/Hydrators/IngredientEntityHydrator.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class IngredientEntityHydrator {
+    private $db;
+    public function __construct(PDO $db) {
+        $this->db = $db;
+}
+    public function getAllIngredientEntities() {
+        $query = $this->db->prepare('SELECT `id`, `name` FROM `ingredients`;');
+        $query->setFetchMode(PDO::FETCH_CLASS, '../IngredientEntity');
```

Comment:
I think this filepath would be wrong? Should it be '../Entities/IngredientEntity'?